app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from shutil import copy
+from os.path import exists
+
+
+class ArgumentException(Exception):
+    pass
```

Comment:
I think its not necessary to create a custom exception. You can use ValueError and then combine em in except block